github-integration-its/src/main/java/org/jenkinsci/plugins/github_integration/SomeTest.java

```diff
@@ -0,0 +1,23 @@
+package org.jenkinsci.plugins.github_integration;
```

Comment:
either it will be under github. that will conflict with too generic github-plugin. Done in the same way as github-branchsource